Google.PowerShell.IntegrationTests/CloudResourceManager/CloudResourceManager.GcProject.Tests.ps1

```diff
@@ -0,0 +1,65 @@
+﻿. $PSScriptRoot\..\GcloudCmdlets.ps1
+Install-GcloudCmdlets
+
+$script:powerShellProjectId = "gcloud-powershell-testing"
+$script:powerShellProjectTwoId = "gcloud-powershell-testing-2"
+$script:powerShellProjectName = "Google Powershell Testing"
+$script:powerShellProjectTwoName = "GCloud PowerShell Testing 2"
+
+Describe "Get-GcIamPolicyBinding" {
+    It "should work" {
+        $projects = Get-GcProject
+        $project = $projects | Where-Object {$_.ProjectId -eq $powerShellProjectId}
```

Comment:
You could write this as 
`$project = $projects | Where ProjectId -eq $powerShellProjectId`
avoiding the curly braces and `$_.` feels cleaner.